### PR TITLE
test(shell): add token theming tests

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:**#7180

## Summary
Add token theming tests for `shell`.

### CODE REVIEW

File: packages/calcite-components/src/components/shell/shell.e2e.ts [hunk 1]
```diff
@@ -122,3 +122,41 @@ describe("calcite-shell", () => {
     expect(panelTop).toBeNull();
   });
 });
+
+describe("slotted overrides", () => {
```
Comment (jcfranco) [inline packages/calcite-components/src/components/shell/shell.e2e.ts:126]: Can you also add a default theme test (for `background`/`border-color` props) and one to cover the deprecated `--calcite-shell-tip-spacing`?

Comment (jcfranco) [inline packages/calcite-components/src/components/shell/shell.e2e.ts:126]: Outer-most describe block is not consistent with [suggested pattern](https://github.com/Esri/calcite-design-system/blob/0956bad28d2693ac2a9b207523878ef66266593f/packages/calcite-components/src/tests/commonTests/themed.ts#L28).

**Review by jcfranco — CHANGES_REQUESTED** [review]

File: packages/calcite-components/src/components/shell/shell.e2e.ts (hunk 1 above)
Comment (Elijbet) [reply packages/calcite-components/src/components/shell/shell.e2e.ts:126]: whoops!

File: packages/calcite-components/src/components/shell/shell.e2e.ts [hunk 2]
```diff
@@ -122,3 +123,60 @@ describe("calcite-shell", () => {
     expect(panelTop).toBeNull();
   });
 });
+
+const tipMangerHTML = html`
```
Comment (jcfranco) [inline packages/calcite-components/src/components/shell/shell.e2e.ts:127]: Typo: `tipMangerHTML` ➡️ `tipManagerHTML`

File: packages/calcite-components/src/components/shell/shell.e2e.ts [hunk 3]
```diff
@@ -122,3 +123,71 @@ describe("calcite-shell", () => {
...
+    <calcite-tip heading="Example tip title">
+      <calcite-link href="http://www.esri.com">An example link</calcite-link>
+    </calcite-tip>
+  </calcite-tip-manager>
+`;
+
+describe("theme", () => {
+  const shellHTML = html`
+    <calcite-shell>
+      ${tipManagerHTML}
```
Comment (jcfranco) [inline packages/calcite-components/src/components/shell/shell.e2e.ts:138]: Nitpick: `tipManagerHTML` is only used once, so it could be inlined here.

**Review by jcfranco — APPROVED** [review]
💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎
💎🐚💎💎💎🐚🐚🐚🐚💎🐚🐚🐚🐚🐚💎🐚💎💎💎🐚💎🐚💎
💎🐚💎💎💎🐚💎💎💎💎💎💎🐚💎💎💎🐚🐚💎🐚🐚💎🐚💎
💎🐚💎💎💎🐚💎🐚🐚💎💎💎🐚💎💎💎🐚💎🐚💎🐚💎🐚💎
💎🐚💎💎💎🐚💎💎🐚💎💎💎🐚💎💎💎🐚💎💎💎🐚💎💎💎
💎🐚🐚🐚💎🐚🐚🐚🐚💎💎💎🐚💎💎💎🐚💎💎💎🐚💎🐚💎
💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎💎

File: packages/calcite-components/src/components/shell/shell.e2e.ts (hunk 3 above)
Comment (Elijbet) [reply packages/calcite-components/src/components/shell/shell.e2e.ts:138]: I'll inline it, although I feel like separating chunks makes it visually easier to parse through html.

**Pull request merged**